napari_laptrack/_function.py

```diff
@@ -8,6 +8,27 @@
 def napari_experimental_provide_function():  # noqa: D103
     return [track_labels_centroid_based]
 
+def _check_and_convert_layers_to_4d(image_layer,labels_layer,viewer):
+    import napari_time_slicer as nts
+    image = image_layer.data
+    labels = labels_layer.data
+    # Deal with input data of various formats and bring both in [T,Z,Y,X] shape/format, maybe with Z=1
+    assert image.shape == labels.shape, "Image and labels must have the same shape."    
+    if len(image) == 2:
```

Comment:
I presume this should be `len(image.shape) == 2` ?

Reply:
Yeah that was a typo, thanks!